.github/workflows/ci.yml

```diff
@@ -34,6 +34,12 @@ jobs:
           channel: 'stable'
       - run: dart --version
       - run: flutter --version
+      - name:
+        run: |
+        cd example
```

Comment:
It should be tabbed inside once. Like below:
```
        run: |
          cd example
```

Reply:
Thanks!